package.json

```diff
@@ -84,27 +85,203 @@
     }
   },
   "contributes": {
+    "icons": {
+      "camel-run": {
+        "description": "Run Camel Application",
+        "default": {
+          "fontPath": "./icons/camel-run.woff",
+          "fontCharacter": "\\E900"
+        }
+      }
+    },
     "commands": [
       {
         "command": "kaoto.open",
-        "title": "Open with Kaoto Graphical Editor for Camel"
+        "title": "Open with Kaoto"
```

Comment:
So we assume that we reach a point where every Camel users will know that Kaoto is providing a graphical editor?
For people that installed Kaoto directly I can imagine yes but for people who installed the extension pack, is it already something that we can assume?